app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,74 @@
+class TweetsController < ApplicationController
+
+require 'pry'
+
+  get '/tweets' do
+    if logged_in?
+      @tweets = Tweet.all
+      erb :'/tweets/index'
+    else
+      redirect to "/login"
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'/tweets/new'
+    else
+      redirect to "/login"
+    end
+  end
+
+  post '/tweets/new' do
```

Comment:
Typically, tweet creation would be posted to `/tweets` instead of `/tweets/new`.